loadbalancer/lbalgorithm.go

```diff
@@ -0,0 +1,82 @@
+package loadbalancer
+
+import (
+	"math/rand"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+)
+
+type lbAlgorithmType int
+
+const (
+	RoundRobin         = "roundrobin"
```

Comment:
style: for me it would be more consistent if it was called "roundRobin"

Reply:
changed